casts/countryPicker/src/UICountryPicker/CountryPicker.tsx

```diff
@@ -121,6 +124,26 @@ export function CountryPicker({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, []);
 
+    React.useEffect(() => {
+        if (isAndroid) {
+            // We should use react-native-android-keyboard-adjust
+            // to fix android keyboard work with UIBottomSheet
+            // also we have to check if component visible
+            // and only if it showing call setAdjustNothing
+            // because in other cases we may need the default adjust - setAdjustPan
+            visible
+                ? AndroidKeyboardAdjust.setAdjustNothing()
+                : AndroidKeyboardAdjust.setAdjustPan();
```

Comment:
Can we somehow find the initial "mode"? Since I'm not sure that we need to return into `pan` mode, I'd rather return into `Resize` (as the most used in Surf)

Reply:
@sertony As I see resize does not work in chat(setAdjustPan work), I will look for something